ckanext/ontario_theme/fanstatic/common.css

```diff
@@ -435,6 +437,516 @@ a:focus {
+  box-shadow: 0 0 0 4px #009adb inset;
+  text-decoration: underline;
+  background-color: #CCCCCC;
+  outline: 4px solid transparent;
+  transition: box-shadow .1s ease-in-out;
+}
+.ontario-navigation li {
+  border-top: 1px solid #ccc;
+  padding: 0;
+}
+.ontario-navigation li:last-of-type{
+  border-bottom:1px solid #ccc
+}
+.ontario-navigation a:hover {
+  text-decoration: underline;
+  background-color: #e6e6e6;
+}
+.ontario-header-button:focus {
+  box-shadow: 0 0 0 4px #009adb;
+  outline: 4px solid transparent;
+  transition: box-shadow .1s ease-in-out;
+}
+.ontario-navigation{
+  position:absolute;
+  width:100%
+}
+/* ===============================
+   Header - media queries
+   =============================== */
+@media screen and (min-width: 40em) {
+  .ontario-application-subheader__heading {
+    font-size: 1.5rem;
+    letter-spacing: 0.0313rem;
+    line-height: 1.5;
+  }
+}
+@media screen and (min-width: 73em) {
+  .ontario-application-subheader{
+    z-index:0
+  }
+  .ontario-navigation--open .ontario-overlay {
+    display: none;
+  }
+  .ontario-navigation {
+    position: relative;
+    max-width: 72rem;
+    width: auto;
+    margin: 0 auto;
+  }
+  .ontario-navigation__container {
+    box-shadow: 0 4px 12px rgb(0 0 0 / 30%);
+    border-radius: 4px;
+    display: none;
+    position: absolute;
+    right: 1rem;
+    top: calc(100% - 0.5rem);
+    transition: none;
+    min-width: 290px;
+    max-width: 30%;
+    z-index: 5;
+  }
+  .ontario-navigation.ontario-navigation--open .ontario-navigation__container {
+    transition: none;
+    display: block;
+  }
+  .ontario-navigation ul {
+    border-bottom: none;
+  }
+}
+/* uses same media query as ontario-only-show-for-small for the first 2 nav tabs in
+the menu dropdowm. Datasets and ministries are displayed in the medium screen nav bar */
+@media screen and (max-width: 0em), screen and (min-width: 40em) {
+  .ontario-navigation__container li:nth-child(-n + 2) {
+    display: none !important;
+  }
+}
+@media screen and (min-width: 40em) and (max-width: 42em){
+  .ontario-header__search-toggler {
+    margin-left: unset !important;
+  }
+}
+/* Moves the closing Menu button when the French title wraps */
+@media screen and (min-width: 40em) and (max-width: 51.5em) {
+  .ontario-header-button--with-outline.fr{
+    top: calc(-4rem + -1.4rem);
+  }
+}
+@media screen and (min-width: 20em) and (max-width: 25.75em){
+  .ontario-header-button--with-outline.fr{
+    top: calc(-4rem + -1rem);
+  }
+}
+@media screen and (min-width: 20em) and (max-width: 21.375em){
+  .ontario-application-subheader-menu__container .ontario-header-button--with-outline{
+    top: calc(-4rem + -1rem);
+  }
+}
+@media screen and (max-width: 73em) {
+  .ontario-navigation--open .ontario-application-subheader-menu__container {
+    position: relative;
+    z-index: 3;
+  }
+  .ontario-navigation a {
+    padding: calc(0.75rem - (0.25rem / 3.25)) 2rem;
+  }
+  .account-text{
```

Comment:
`account-text` does not seem to be called in the application for the admin menu items. Actually removing both classes `class="text account-text"` from the span in `header.html` (`{% block header_account_container_content %}`) doesn't change the appearance of the texts.